modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -149,7 +149,7 @@ spec:
 
 [NOTE]
 ====
-In OADP 1.4, the `kopia` will become the default `uploaderType` value.
+In OADP 1.4, `kopia` will become the default `uploaderType` value.
```

Comment:
Not a merge blocker, but please consider creating a follow up PR to reword this sentence. The [ISG states](https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations) that we should avoid claims about future product releases. 

```suggestion
In a future OADP release, it is planned that the `kopia` tool will become the default `uploaderType` value.
```

Reply:
This is the format that PX has asked us to use when talking about plans for future releases.